cost/aws/reserved_instances/recommendations/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v2.18
+
+- Renamed `Reserved Instance` policy_set from info block to `Reserved Instances`.
+- Added `resourceType` incident field.
```

Comment:
Should be reworded to be more sensible to users that don't know about the policy template language. In this case, refer to the incident field by the name the user would see (Resource Type)

(This applies to the rest of the items on the list as well)

Reply:
Renamed incident fields to what the user will see!